tokens.json

```diff
@@ -1219,5 +1219,16 @@
       "twitter": "https://twitter.com/Jetplane_NFT",
       "discord": "https://discord.gg/jetplane"
     }
+  },
+  "df1d850c46d6c9d12cbf6181c35db9225a91b77c8a646b7f636f8ae4": {
+    "project": "Danketsu Token $NINJAZ",
+    "categories": ["Gamefi", "NFT", "Launchpad", "Bridge"],
```

Comment:
@vivid4516 It has to be `GameFi`. It's case sensitive. Please run `npm i && npm run test` before you pushing your changes